proxy.py

```
@@ -2316,12 +2316,21 @@ async def list_models_openai(auth: AuthContext = Depends(verify_api_key)):
-    return {"object": "list", "data": local_entries + registry_only}
+    registry_only = []
+    alias_entries = []
+    for name, cap in registry.items():
+        if name in ollama_set:
+            continue
+        if "alias" in (cap.tags or []):
+            alias_entries.append({
+                "id": name,
+                "object": "model",
+                "owned_by": "llm-relay-alias",
+                "description": f"Alias → routed to best available local model",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)